src/components/Filter.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import {FormControlLabel, Checkbox, Button, Drawer, Divider, IconButton} from '@mui/material';
+import {useState} from 'react';
+import {RangeSlider} from '../Helper_components/RangeSlider';
+
+
+
+
+const listOfGenres = [
```

Comment:
Why do we sort hardcoded values?
Let's sort this ourselves, since this is not dynamic.

Reply:
Thanks, I fixed this.

![Screenshot_5](https://user-images.githubusercontent.com/70068774/171999435-60baa651-43aa-43c6-b007-9db0b7ff626f.png)